requirements.txt

```diff
@@ -1 +1,11 @@
 aiohttp==1.0.0
+aionotify==0.2.0
```

Comment:
You don't really have to touch this file (actually I'm gonna remove it, it's not used anyway).
Dependencies are set in install_requires section of setup.py, you cann add paco and aionotify there.